examples/hosts/app-integration/external-data/src/externalData.ts

```diff
@@ -6,21 +6,30 @@
 import type { IEvent } from "@fluidframework/common-definitions";
 import { TypedEventEmitter } from "@fluidframework/common-utils";
 
+/**
+ * string-parsed representation of task data, returned from {@link parseStringData}.
+ */
+interface ParsedTaskData {
```

Comment:
Probably export this (since parseStringData is exported, and this interface is required to fully describe its signature)?